From 43b399e20c523c45f5c34e2156689c57201f94f3 Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@turris.com>
Date: Mon, 22 Sep 2025 09:00:43 +0200
Subject: [PATCH] qr_details: Fix the title

Make sure that the title in Wi-Fi details states correctly whether it is
LAN Wi-Fi or a Guest Wi-Fi.
---
 ui/pages/qr_details.slint | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ui/pages/qr_details.slint b/ui/pages/qr_details.slint
index dab8dbd..852b10e 100644
--- a/ui/pages/qr_details.slint
+++ b/ui/pages/qr_details.slint
@@ -26,7 +26,7 @@ export component QRDetails inherits Page {
 
         VerticalLayout {
             PageTitle {
-                text: "Guest Wi-Fi";
+                text: (Readouts.initial-setup-finished ? "Guest Wi-Fi" : "LAN Wi-Fi");
             }
 
             VerticalLayout {
-- 
2.51.0

